utils/validate-install-plan.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+const path = require('path');
+const {
+  fetchPaginatedGHResults,
+  filterQuickstartConfigFiles,
+  filterOutTestFiles,
+} = require('./github-api-helpers');
+const {
+  findMainInstallConfigFiles,
+  readQuickstartFile,
+  pathsToSanitize,
+} = require('./helpers');
+
+const url = pathsToSanitize[0];
+
+/**
+ * Gets all install plain ids under installs/ dir
+ * @returns {Array} Array of unique install plan ids
+ */
+const getAllInstallPlanIds = () => {
+  return findMainInstallConfigFiles().reduce((acc, filePath) => {
+    const { contents } = readQuickstartFile(filePath);
+
+    const { id } = contents[0];
+    return [...new Set([...acc, id])];
+  }, []);
+};
+
+/**
+ * Gets all the install plans and paths for an array of config files
+ * @param {Array} configFiles - Array of config files
+ * @returns {{filePath: string, installPlans: Array}[]} Array of paths and install plans for file
+ */
+const getConfigInstallPlans = (configFiles) => {
+  return configFiles.map(({ filename }) => {
+    const filePath = path.join(process.cwd(), `../${filename}`);
+    const { installPlans } = readQuickstartFile(filePath).contents[0];
+
+    return { filePath, installPlans };
+  });
+};
+
+/**
+ * Gets all the install plans and paths for an array of config files
+ * @param {{filePath, installPlans}[]} configInstallPlanFiles - Array of objects with path and install plans for file
+ * @param {String[]} installPlanIds - Array of install plan ids
+ * @returns {{filePath, installPlans}[]} Array of paths and install plans for file
+ */
+const getInstallPlansNoMatches = (configInstallPlanFiles, installPlanIds) => {
+  return configInstallPlanFiles
+    .map(({ installPlans, filePath }) => {
+      const nonExistentInstallPlans = installPlans.filter(
+        (plan) => !installPlanIds.includes(plan)
+      );
+      return { filePath, installPlans: nonExistentInstallPlans };
+    })
+    .filter(({ installPlans }) => installPlans.length > 0);
+};
+
+/**
+ * Gets all the install plans and paths for an array of config files
```

Comment:
Same here, description has been copy/pasted so just needs to be changed

Reply:
Got this over on the main fork PR here: 823dc76